pkg/controllers/state/statenode.go

```diff
@@ -45,23 +47,25 @@ type StateNodes []*StateNode
 
 // Active filters StateNodes that are not in a MarkedForDeletion state
 func (n StateNodes) Active() StateNodes {
-	return lo.Filter(n, func(node *StateNode, _ int) bool {
+	n = lo.Filter(n, func(node *StateNode, _ int) bool {
 		return !node.MarkedForDeletion()
 	})
+	return n
 }
 
 // Deleting filters StateNodes that are in a MarkedForDeletion state
 func (n StateNodes) Deleting() StateNodes {
-	return lo.Filter(n, func(node *StateNode, _ int) bool {
+	n = lo.Filter(n, func(node *StateNode, _ int) bool {
 		return node.MarkedForDeletion()
 	})
+	return n
 }
 
 // Pods gets the pods assigned to all StateNodes based on the kubernetes api-server bindings
-func (n StateNodes) Pods(ctx context.Context, c client.Client) ([]*v1.Pod, error) {
+func (n StateNodes) Pods(ctx context.Context, kubeClient client.Client) ([]*v1.Pod, error) {
```

Comment:
This is fine, but I think this may have been updated from moving things around and the rename may not be required here

Reply:
Fair, just renamed them all the kubeClient anyway to make them the same across all functions

Reply:
Sounds fine 👍🏼